redaxo/src/core/functions/function_rex_other.php

```diff
@@ -63,3 +63,48 @@ function rex_ini_get($varname)
 
     return $val;
 }
+
+/**
+ * @internal
+ */
+function packages_title()
```

Comment:
Mir gefällt es nicht so, hier eine globale Funktion dafür zu haben.
Nach unserem bisherigen Schema fände ich besser eine page oberdatei für die drei subpages, die den titel erzeugt, und dann die unterpage einbindet.
Also zwischen der Haupt packages.php und den drei Subpages.

Reply:
Dir gehts drumm in welcher datei die function hier definiert ist?

Reply:
Ich finde es ungewohnt für uns, überhaupt hier mit einer Funktion zu arbeiten.
Und sehe aktuell auch nicht die Notwendigkeit, wenn es eine Basis-Page für die drei Subpages gäbe, die bereits den Titel ausgeben würde. So machen wir es doch sonst auch immer.
Sowas: https://github.com/redaxo/redaxo/blob/master/redaxo/src/core/pages/system.php

Reply:
D.h. Du hättest gerne eine neue methode statt einer funktion?

Irgendwo muss dir logik ja hin. Und mit den generischen methoden alleine welche wir haben alleine ist die logik nicht umsetzbar

Reply:
Ich passe es so an, wie ich es meine.

Reply:
So meinte ich es: https://github.com/redaxo/redaxo/pull/4264/commits/390cbb3deab142fb455e23c1b70c51dc367c2487
`packages.details.php` ist nun die "Ober"-Page für die drei Subpages, die den Rahmen setzen (Überschrift, Navi).